discord/cache.py

```diff
@@ -1,122 +1,46 @@
 from __future__ import annotations
 
-from typing import TYPE_CHECKING, Any, OrderedDict
+from typing import TYPE_CHECKING, Any, Dict, OrderedDict
+
+from .types.snowflake import Snowflake
+
 
 if TYPE_CHECKING:
 
     from .api.httphandler import HTTPHandler
     from .guild import Guild
     from .message import Message
     from .role import Role
-    from .types.snowflake import Snowflake
     from .user.member import Member
     from .user.user import User
 
 
-class LFUCache:
-
-    capacity: int
-    _cache: OrderedDict[Snowflake, Any]
-    _frequency: dict[Snowflake, int]
-    httphandler: HTTPHandler
-
+class LFUCache(OrderedDict):
```

Comment:
I heard UnorderedDict increases performance

Reply:
What is `UnorderedDict`?

Reply:
I initially thought it was the same as `dict` but apparently it's not.
Guess we gotta work on one ourselves